tests/components/homematicip_cloud/test_config_flow.py

```diff
@@ -15,6 +15,7 @@
     }
     flow = config_flow.HomematicipCloudFlowHandler()
```

Comment:
Please rewrite the tests to use the core interfaces. See hue config flow tests for good examples. Then we can remove this.

Reply:
removed all HomematicipCloudFlowHandler